docs/glossary.md

```diff
@@ -57,7 +57,7 @@ To access resources such as SGTS services and GCC via this device, you must onbo
 
 </a>
 
-If you are using a non-SE GSIB device, every time you log in to your device, you will be prompted to enter your BitLocker PIN. You can access the internet and intranet using this device.
+If you are using a non-SE GSIB device, every time you log in to your device, you are prompted to enter your BitLocker PIN. You can access the internet and intranet using this device.
 
 You can't onboard a GSIB device to SEED.
```

Comment:
"You cannot onboard this device to SEED" ? what do you think?

Reply:
done.